### PR TITLE
chore: prep

### PR DESCRIPTION
**Pull request by kevcodez**

### CODE REVIEW

File: apps/studio/components/interfaces/Billing/SpendCapModal.tsx [hunk 1]
```diff
@@ -18,138 +27,95 @@ const SpendCapModal = ({ visible, onHide }: SpendCapModalProps) => {
             limits, but any extra usage beyond the plan's limit will be charged per usage.
           </p>
           <p className="text-sm">
-            Take a look at the following table to see which resources are chargeable and how they
-            are charged:
+            Launching additional projects or enabling project add-ons will incur additional monthly
+            fees indepdent of your spend cap.
```
Comment (ecktoteckto) [inline apps/studio/components/interfaces/Billing/SpendCapModal.tsx:31]:
```suggestion
            fees independent of your spend cap.
```

File: apps/docs/content/guides/platform/org-based-billing.mdx [hunk 2]
```diff
@@ -37,25 +46,28 @@ It is not possible to mix different plans inside a single organization. You cann
...
-| Function Count             | 25                       | 500/1000                                    | Unlimited  |
-| Storage Images Transformed | Unavailable on Free Plan | 100 included, then $5 per 1000              | Custom     |
-| Realtime Message Count     | 2 million                | 5 million included, then $2.5 per million   | Custom     |
-| Realtime Peak Connections  | 200                      | 500 included, then $10 per 1000             | Custom     |
-
-## Usage-based billing for compute
+| Metric                           | Free                     | Pro/Team                                          | Enterprise |
+| -------------------------------- | ------------------------ | ------------------------------------------------- | ---------- |
+| Egress                           | 5GB                      | 250GB included, then $0.09 per GB                 | Custom     |
+| Database Size                    | 500MB                    | 8GB disk per project included, then $0.125 per GB | Custom     |
```
Comment (pamelachia) [inline apps/docs/content/guides/platform/org-based-billing.mdx:52]: disk size?

File: apps/docs/content/guides/platform/org-based-billing.mdx [hunk 3]
```diff
@@ -78,33 +90,43 @@ We only count compute hours for instances that are active. Paused projects do no
...
-
-### Compute credits
+### Compute Credits
 
 Paid plans come with $10 of Compute Credits to cover one Micro instance or parts of any other [Compute Add-On](/docs/guides/platform/compute-add-ons).
 
-Compute Credits are deducted from your Compute Usage. You can launch as many instances as you want on paid plans and we'll bill based on the compute hours. If you upgrade an instance for 24 hours, you'll only be billed for those 24 hours of additional compute. Compute hours are billed when you do plan up/downgrades (resets the billing cycle) or whenever your billing cycle resets once a month.
+Compute Credits are deducted from your Compute Usage. You can launch as many instances as you want on paid plans and we'll bill based on the compute hours.
 
 Compute Credits do not apply to other usage fees like egress, database size, ... They solely apply to Compute usage.
```
Comment (pamelachia) [inline apps/docs/content/guides/platform/org-based-billing.mdx:99]: disk size here too

File: apps/docs/content/guides/platform/project-transfer.mdx [hunk 4]
```diff
@@ -38,14 +38,13 @@ Target organization - the organization you want to move the project to
...
+The target organization will be charged at the end of the billing cycle for the remaining time in the billing cycle.
+As the billing cycle from the source and target organization likely differ, the amounts can vary.
 
-If you’re using a [Compute Add-On](/docs/guides/platform/compute-add-ons), the source organization will get credits for the remaining unused time, whereas the target organization won’t have any immediate charges.
-As we’re charging compute usage-based with [Organization-based Billing](/docs/guides/platform/org-based-billing), you’ll be charged for the usage when the next billing cycle hits based on how long the project has been in the target organization.
+All project add-ons are charged usage-based at the end of your billing cycle, see [How billing works](/docs/guides/platform/org-based-billing).
 
 Similarly, for other usage metrics such as database size, egress, image transformations and so on, the usage until the project transfer will be billed on the source organization and future usage will be added to the target organization once moved.
```
Comment (pamelachia) [inline apps/docs/content/guides/platform/project-transfer.mdx:49]: disk size

**Review by pamelachia — COMMENTED** [review]
some places i read that had database size where it should be disk size. Feel free to not change anything if I'm mistaken.

**Comment by roryw10** [comment]
:)

**Review by roryw10 — APPROVED** [review]

**Review by alaister — APPROVED** [review]
Approving on behalf of dashboard

**Pull request merged**